src/components/Book.js

```diff
@@ -1,14 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { useDispatch } from 'react-redux';
-import { deleteBook } from '../redux/books/books';
 
 const Book = (props) => {
-  const {
-    id, title, author, category,
-  } = props;
-
-  const dispatch = useDispatch();
+  const { category, title, author } = props;
```

Comment:
- [ ] According to the project you should  Fetch your books from Bookstore API on when the component has rendered with` useEffect` . Please fix it 👍